assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -308,6 +308,8 @@ describe( 'modules/analytics properties', () => {
 		} );
 
 		describe( 'getPropertiesIncludingGA4', () => {
+			it.todo( 'returns undefined if UA or GA properties are loading' );
```

Comment:
Happy to flesh this out! (or create a ticket TODO this)

Reply:
Yes, let's create a test for it in the scope of this ticket.